### PR TITLE
Jf feature onboarding

### PR DESCRIPTION
**Pull request by frank920120**
Create onboarding screen

### CODE REVIEW

File: client/js/components/OnboardingOne/styles.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { StyleSheet } from "react-native";
+
+export const styles = StyleSheet.create({
+  Container: { flex: 1 },
+  OnboardingBanner: {
+    width: "100%",
+    resizeMode: "contain",
+    alignItems: "center",
+    justifyContent: "center",
+    marginTop: "10%"
```
Comment (siddharthparmar7) [inline client/js/components/OnboardingOne/styles.js]: Use Dimensions.

File: client/js/components/OnboardingThree/OnboardingThree.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+import { View, Text, Image, TouchableOpacity } from "react-native";
+import { styles } from "./styles";
+const OnboardingThree = ({ navigation }) => {
+  const [isRegisterActive, setIsRegisterActive] = useState(false);
+  const [isSignInActive, setIsSignInActive] = useState(false);
+
+  const handleReisterToggle = () => {
+    setIsRegisterActive(!isRegisterActive);
+    setIsSignInActive(false);
+    // navigation.navigate("Layout");
```
Comment (siddharthparmar7) [inline client/js/components/OnboardingThree/OnboardingThree.js]: remove commented code. Thank you.

File: client/js/components/OnboardingThree/styles.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+  root: {
+    flex: 1,
+    alignItems: "center"
+  },
+  logo: {
+    resizeMode: "contain",
+    width: "100%",
+    maxWidth: 300,
+    marginTop: "20%",
+    marginBottom: "10%"
```
Comment (siddharthparmar7) [inline client/js/components/OnboardingThree/styles.js]: Please use Dimensions.

File: client/js/components/OnboardingTwo/styles.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
+import { StyleSheet } from "react-native";
+
+export const styles = StyleSheet.create({
+  Container: { flex: 1 },
+  OnboardingBanner: {
+    width: "100%",
+    resizeMode: "contain",
+    alignItems: "center",
+    justifyContent: "center",
+    marginTop: "10%"
```
Comment (siddharthparmar7) [inline client/js/components/OnboardingTwo/styles.js]: Please use Dimensions.

File: client/js/screens/Onboarding/Onboarding.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { View, Text } from "react-native";
+import { styles } from "./styles";
+import Swiper from "react-native-swiper";
+import OnboardingOne from "../../components/OnboardingOne";
+import OnboardingTwo from "../../components/OnboardingTwo";
+import OnboardingThree from "../../components/OnboardingThree";
+const Onboarding = ({ navigation }) => {
+  return (
+    <View style={{ flex: 1 }}>
```
Comment (siddharthparmar7) [inline client/js/screens/Onboarding/Onboarding.js]: Please move your styles to styles.js file

File: yarn.lock [hunk 6]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
```
Comment (siddharthparmar7) [inline yarn.lock]: remove this file if not needed.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Please make changes.

File: client/js/components/OnboardingThree/OnboardingThree.js [hunk 7]
```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { View, Text, Image, TouchableOpacity } from "react-native";
+import { styles } from "./styles";
+const OnboardingThree = ({ navigation }) => {
+  const handleReisterClick = () => {
+    navigation.navigate("Layout");
+  };
+  const handleSignInClick = () => {
+    navigation.navigate("Layout");
+  };
```
Comment (siddharthparmar7) [inline client/js/components/OnboardingThree/OnboardingThree.js]: Both of these functions have the same logic. Please make just one function to help you navigate to Layout.

File: client/js/screens/Onboarding/styles.js [hunk 8]
```diff
@@ -0,0 +1,24 @@
+import { StyleSheet } from "react-native";
+
+export const styles = StyleSheet.create({
+  root: {
+    flex: 1
+  },
+  dot: {
```
Comment (siddharthparmar7) [inline client/js/screens/Onboarding/styles.js]: what is a dot?

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Please make the changes as per our discussion. Thank you.

File: client/js/screens/Onboarding/Onboarding.js [hunk 9]
```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { View } from "react-native";
+import { styles } from "./styles";
+import Swiper from "react-native-swiper";
+import { OnboardingData } from "../../helpers/onboarding";
+import OnboardingIntro from "../../components/OnboardingIntro";
+import OnboardingAuth from "../../components/OnboardingAuth";
+console.log(OnboardingData);
```
Comment (siddharthparmar7) [inline client/js/screens/Onboarding/Onboarding.js]: remove this console log.

File: client/js/components/OnboardingIntro/styles.js [hunk 10]
```diff
@@ -0,0 +1,25 @@
+import { StyleSheet, Dimensions } from "react-native";
+const { width, height } = Dimensions.get("window");
```
Comment (teelau) [inline client/js/components/OnboardingIntro/styles.js:2]: `width` and `height` are keywords. Name them something else like screenWidth

**Review by teelau — CHANGES_REQUESTED** [review]

**Review by siddharthparmar7 — APPROVED** [review]

File: client/js/components/OnboardingIntro/styles.js (hunk 10 above)
Comment (siddharthparmar7) [reply client/js/components/OnboardingIntro/styles.js:2]: That's a good idea, Tommy.

**Pull request merged**